topology/University of Chicago/UChicago/UChicago_Open_Science_CE.yaml

```diff
@@ -0,0 +1,61 @@
+# TODO Change this to "true" once this is truly up and running.
+Production: false
+SupportCenter: Self Supported
+
+GroupDescription: These are the Open Science CE resources hosted at UChicago.
+
+GroupID: 533
+
+# Resources contains one or more resources in this
+# ResourceGroup. A resource provides one or more services
+Resources:
+  UChicago_Open_Science_CE1:
+    Active: false
+    Description: The first Open Science CE - a combination CE and flocking submit host
+    ID: 1049
+    ContactLists:
+      # Administrative Contact is one to three people to contact regarding administrative issues
+      Administrative Contact:
+        Primary:
+          Name: Matyas Selmeci
+          ID: ec1013224934d6a11a2a46a5234b3337095f5ec4
+        Secondary:
+          Name: Brian Lin
+          ID: 026718919cb6a13e64c8df3b53398c7214ceeaf1
+        Tertiary:
+          Name: Brian Bockelman
+          ID: 66ee5cfb622a7343dac85dee42815d1f4fbc2d85
+
+      Security Contact:
+        Primary:
+          Name: Matyas Selmeci
+          ID: ec1013224934d6a11a2a46a5234b3337095f5ec4
+        Secondary:
+          Name: Brian Lin
+          ID: 026718919cb6a13e64c8df3b53398c7214ceeaf1
+        Tertiary:
+          Name: Brian Bockelman
+          ID: 66ee5cfb622a7343dac85dee42815d1f4fbc2d85
+
+    FQDN: ce1.opensciencegrid.org
+    ### FQDNAliases (optional) are any other DNS aliases by which this host can be accessed
+    # FQDNAliases:
+    #   - <HOSTNAME1>
+    #   - <HOSTNAME2>
+
+    # Services is one or more services provided by this resource;
+    # valid services are listed in topology/services.yaml with the format "<SERVICE NAME>: <ID>"
+    Services:
+      CE:
+        Description: The CE part of this Open Science CE
+        Details:
+          hidden: false
+
+      Submit Node:
+        Description: The submit part of this Open Science CE
+        Details:
+          hidden: false
+
+    VOOwnership:
+      OSG: 100
```

Comment:
I don't think we ever want to set `VOOwnership` to OSG since accounting records are compared to VOOwnership to determine opportunistic usage. In this case, any OSG VO pilots running through the CE will not be considered opportunistic since OSG owns the resource. It may not actually matter if we're disabling accounting on the CE portion but we should do the right thing here.

Reply:
Gotcha. I'm pretty sure VOOwnership is optional.